2023-07-08 13:27:40,897 ERROR 'Pedestrian' object has no attribute 'pedestrian_no'
Traceback (most recent call last):
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\game.py", line 9, in <module>
    game_loop.looping()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\game_loop.py", line 121, in looping
    self._02_level.update(dt)
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_02_level\_02_main.py", line 386, in update
    self.update_pedestrians_timers()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_02_level\_02_main.py", line 337, in update_pedestrians_timers
    pedestrians_timer.update()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_02_level\pedestrian.py", line 195, in update
    self.spawn_pedestrian()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_02_level\pedestrian.py", line 181, in spawn_pedestrian
    Pedestrian(groups=self.groups,
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_02_level\pedestrian.py", line 35, in __init__
    super().__init__(groups=groups,
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_02_level\entity.py", line 19, in __init__
    self.set_hitbox(pos)
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_02_level\pedestrian.py", line 59, in set_hitbox
    if self.pedestrian_no == 1 or self.pedestrian_no == 2:
       ^^^^^^^^^^^^^^^^^^
AttributeError: 'Pedestrian' object has no attribute 'pedestrian_no'
